alf/examples/carla_conf.py

```diff
@@ -66,3 +86,21 @@ def _make_simple_preproc(spec, use_bias=False):
                 prev_action_spec,
                 use_bias='camera' not in observation_spec['observation']),
     }
+
+
+def create_input_preprocessor_masks():
+    # create input mask for flexibily mask out some sensors
+    observation_spec = alf.get_observation_spec()
+    observation_preprocessor_masks = {}
+    for sensor, spec in observation_spec['observation'].items():
+        if sensor in [
+                'gnss',
+                'goal',
+                'radar',
+        ]:  # mask selected seneors out of observation for training
+            print("-----skip {}".format(sensor))
```

Comment:
Is this intended or for debugging purpose only?

Reply:
This is an intended message on the skipped sensor. Changed to logging instead.